### ISSUE TITLE
Check to see if localtime exists before copy - causes provision base to fail

### ISSUE THREAD

**Issue by sn3ak** [msimerson/Mail-Toaster-6#368]
### Expected behavior
provision base completes successfully without error

### Observed behavior
provision base fails with the following:
   ***   setting base jail timezone (to hosts)   ***
                                               
cp: /etc/localtime: No such file or directory  

### Steps to reproduce
sudo rm /etc/localtime
provision base

### Extra Notes
Rent a FreeBSD VM from a popular hosting company who doesn't include tzsetup as one of their init tasks. I've been bit by this once before but forgot before starting again.

### Suggested fix
```sh
if [ -f /etc/localhost ]
	cp /etc/localtime "$BASE_MNT/etc" || exit
else
	echo Please run tzsetup and run provision base again
	exit
fi
```

I bet you have a better way though.

Thanks!

**Comment by msimerson** [comment]
Hmm, one clever solution would be using a GeoIP lookup to "discover" the correct TZ and set it automatically. Another simpler one would be to install a GMT one, which is a reasonable default for a system without the TZ set up. Thoughts?

**Comment by sn3ak** [comment]
The GeoIP which you setup for the self signed certificate is great, but I do like the simplicity of just setting it to GMT. I wonder if it would be worth it to echo a message such as "tzsetup not previously ran, using GMT for localtime" ? This of course could also be solved by adding a note to the Wiki to make sure it's configured as part of the FreeBSD/prep OS step.

**Comment by msimerson** [comment]
Right, it belongs in host setup... So something more like this:

```diff
diff --git a/provision-host.sh b/provision-host.sh
index ce3b10e..5ad4f3a 100755
--- a/provision-host.sh
+++ b/provision-host.sh
@@ -272,6 +272,13 @@ install_sshguard()
        # service sshguard start
 }
 
+check_timezone()
+{
+       if [ ! -e "/etc/localtime" ]; then
+               tzsetup || exit
+       fi
+}
+
 check_global_listeners()
 {
        tell_status "checking for host listeners on all IPs"
@@ -576,6 +583,7 @@ update_host() {
        configure_etc_hosts
        configure_csh_shell ""
        configure_bourne_shell ""
+       check_timezone
        check_global_listeners
        echo; echo "Success! Your host is ready to install Mail Toaster 6!"; echo
 }
```

**Comment by sn3ak** [comment]
That seems like a good solution. Thank you as always!

**Comment by msimerson** [comment]
I've included that code in #370 . If you get a chance to test it that'd be great.

**Comment by sn3ak** [comment]
I'll do some testing on it tomorrow.
